### ISSUE TITLE
Able to access /secure route without actually logging in

### ISSUE THREAD

**Issue by noahjstewart** [nuxt-community/auth-module#259]
With Nuxt being an SSR framework, all pages are given to the client upon first load, even the content of the /secure page. I am able to easily view this page by typing `window.__NUXT__.state.auth.loggedIn = true` 

Is there a workaround for this, so that I can have protected routes on the server without giving the content of them to the client on the first load?

<div align="right"><sub><em>This question is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/auth-module/issues/c217">#c217</a>)</em></sub></div>

**Comment by manniL** [comment]
@stewie0013 Be aware that people **always** will have access to the front-end code, no matter how good you try to hide it.

That's why the *secret* part should be inside your API, which should validate the user on request.

**TL;DR: No**
